packages/playground/src/components/funds_card.vue

```diff
@@ -1,15 +1,24 @@
 <template>
-  <v-btn
-    id="tftBtn"
-    width="2000"
-    color="white"
-    @click="addTFT"
-    class="px-lg-6 px-md-2 px-sm-0 mx-sm-0"
-    style="color: white; max-width: 140px; width: auto; background-color: var(--primary)"
-    :loading="loadingAddTFT"
-  >
-    GET TFT
-  </v-btn>
+  <v-tooltip location="bottom" v-if="showTooltip()" close-delay="1000" color="primary">
+    <template #activator="{ props }">
+      <v-btn
+        id="tftBtn"
+        width="2000"
+        color="white"
+        @click="addTFT"
+        class="px-lg-6 px-md-2 px-sm-0 mx-sm-0"
+        style="color: white; max-width: 140px; width: auto; background-color: var(--primary)"
+        :loading="loadingAddTFT"
+        v-bind="props"
+      >
+        GET TFT
+      </v-btn>
+    </template>
+    <div>
+      Get TFT using BTC or Credit card to Stellar bridge, then use bridge to get your TFT from Stellar.
```

Comment:
Get TFT on Stellar using BTC or Credit card, then you can bridge it to your tfchain account using the Bridge in the dashboard